guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -39,19 +38,20 @@ https://projects.theforeman.org/issues/38829
 [id="katello-upgrade-warnings"]
 == Upgrade warnings
 
-=== Structured APT migration for deb content
+=== Structured APT migration for Deb content
 
-For users with deb type content, it is recommended to migrate to "structured APT" enabled mode *before* upgrading. Instructions for migrating on Katello <= 4.18 can be found here: https://docs.theforeman.org/3.16/Managing_Content/index-katello.html#enabling-structured-apt-content
-If you do not migrate your deb content before upgrading, it will be automatically migrated by the foreman-installer run after upgrading.
-However, if this automatic migration run is omitted, or fails, some of your deb type content may be left in an inconsistent state.
+For users with Deb type content, {Team} recommends that you to migrate to "structured APT" enabled mode before upgrading {Project}.
+If you do not migrate your Deb content before upgrading, it will be automatically migrated by the foreman-installer run after upgrading.
+However, if this automatic migration run is omitted or fails, your Deb type content may be left in an inconsistent state.
 You will also need to perform a complete sync of your smart proxies after the upgrade.
-This is why we recommend migrating before the upgrade, at which point you can easily re-run the migration or get help in case there is an issue.
+Detailed instructions for how to migrate before upgrading to Katello 4.19 can be found here:
+https://docs.theforeman.org/3.16/Managing_Content/index-katello.html#enabling-structured-apt-content
 
 [id="katello-deprecations"]
 == Deprecations
 
 The deprecated `auto_attach` and `autoheal` parameters have been removed from all API endpoints.
 These options are no longer relevant because all organizations are SCA-only, and attaching subscriptions to hosts and activation keys is obsolete.
 
-With the switch to structured APT for deb content, new publications for deb content will no longer add metadata under the `dists/default/` path.
-Use the metadata published under the paths used by the remote repository being synced from instead.
+With the switch to structured APT for Deb content, new publications for Deb content will no longer add metadata under the `dists/default/` path.
```

Comment:
Above, you wrote "... publish a `Release` file ...". I would use the same wording here too.

Reply:
I made it the more neutral "metadata" in both places.